dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -18,7 +18,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 # The file referenced (`/usr/share/dpkg/no-pie-compile.specs`) is in the `libdpkg-perl` package on Debian
 RUN apt-get update   && \
     apt-get install -y  \
-        curl            \
+        libpci3 libsensors5 \
```

Comment:
why adding these two packages? who needs them?

Reply:
```
Unpacking libsnmp30:amd64 (5.7.3+dfsg-1.5) ...
dpkg: dependency problems prevent configuration of libsnmp30:amd64:
 libsnmp30:amd64 depends on libpci3 (>= 1:3.5.2-1); however:
  Package libpci3 is not installed.
 libsnmp30:amd64 depends on libsensors5 (>= 1:3.5.0); however:
  Package libsensors5 is not installed.
```
It is really not required because it gets added due to "apt-get -y install -f". 
Will remove.